modules/models/src/main/java/org/eclipse/xpanse/modules/models/constant/LoggingKeyConstant.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+package org.eclipse.xpanse.modules.models.constant;
+
+/**
+ * Constant keys for logging.
+ */
+public class LoggingKeyConstant {
```

Comment:
would prefer to keep this class in the logging module.

Reply:
Moved